features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -888,6 +888,7 @@ CREATE TABLE IF NOT EXISTS IDN_OIDC_REQ_OBJ_CLAIM_VALUES (
 
 CREATE TABLE IF NOT EXISTS IDN_CERTIFICATE (
             ID INTEGER NOT NULL AUTO_INCREMENT,
+            UUID CHAR(36),
```

Comment:
This diff ideally should not be occurred since that is already added by https://github.com/wso2/carbon-identity-framework/pull/6085